src/Calculator.Shared/Views/MainPage.xaml

```diff
@@ -61,7 +62,8 @@
 		</Style>
 	</Page.Resources>
 
-	<Grid toolkit:VisibleBoundsPadding.PaddingMask="All">
+	<Grid toolkit:VisibleBoundsPadding.PaddingMask="All"
+		  android:Background="White">
```

Comment:
why background only for android?

Reply:
Because on android the splashscreen is used as the background but we want to keep the frosting effect on windows